src/Configuration/Dotenv.hs

```diff
@@ -103,19 +109,35 @@ applySetting kv@(k, v) = do
 -- environment.
 info :: MonadIO m => (String, String) -> DotEnv m ()
 info (key, value) = do
-  Config { .. } <- ask
-  when configVerbose
-    $ liftReaderT . liftIO
-    $ putStrLn
-    $ "[INFO]: Load env '" ++ key ++ "' with value '" ++ value ++ "'"
+  Config {..} <- ask
+  when configVerbose $
+    liftReaderT . liftIO $
+    putStrLn $ "[INFO]: Load env '" ++ key ++ "' with value '" ++ value ++ "'"
 
 -- | The helper allows to avoid exceptions in the case of missing files and
 -- perform some action instead.
 --
 -- @since 0.3.1.0
-onMissingFile
-  :: MonadCatch m
+onMissingFile ::
+     MonadCatch m
   => m a -- ^ Action to perform that may fail because of missing file
-  -> m a               -- ^ Action to perform if file is indeed missing
+  -> m a -- ^ Action to perform if file is indeed missing
   -> m a
 onMissingFile f h = catchIf isDoesNotExistError f (const h)
+
+-- | The helper throws an exception if the allow duplicate is set to False.
+onForbidDuplicates :: MonadIO m => String -> m ()
+onForbidDuplicates key =
```

Comment:
@FranzGB The name suggests that this is a callback. However, I see it as an standard function. Let's rename it to `forbidDuplicates`. Does that sound good?

Reply:
For sure